### ISSUE TITLE
thousands of bookmarks.xbel files

### ISSUE THREAD

**Issue by teodias** [floccusaddon/floccus#1666]
### Which version of floccus are you using?

5.2.3

### How many bookmarks do you have, roughly?

around 1000 bookmarks more or less
### Are you using other means to sync bookmarks in parallel to floccus?

no, floccus is the only synchronization used

### Sync method

Google Drive

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

edge, chrome, firefox, opera, ios app

### Which version of Nextcloud Bookmarks are you using? (if relevant)

-

### Which version of Nextcloud? (if relevant)

-

### What kind of WebDAV server are you using? (if relevant)

-

### Describe the Bug

some days ago realized having a lot of .xbel files, deleted the ones i saw. then figured out googldrive is paging the list and after scrolling down more and more files came up. stopped counting them after 1000 found .xbel files.

### Expected Behavior

all browser extensions and ios safari apps sync to specified dedicated .xbel file. i run 2 profiles, one auto sync for all devices and browsers and one manual sync for a master file as backup. wasnt the badest idea to have a master backup as we now see?

### To Reproduce

install a browser of your chaoice. install the apropriate extension. run an autosync profile for some time.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by teodias** [comment]
after fixing this issue, please provide also a cleaning up solution to get rid of those files

**Comment by marcelklehr** [comment]
Thank you for reporting this. This is a duplicate of #1652

**Comment by teodias** [comment]
you will add the request for a cleaning routine to the master ticket?
